### ISSUE TITLE
Invalid requests sent out for voxel content

### ISSUE THREAD

**Issue by javagl** [CesiumGS/cesium#12001]
### What happened?

CesiumJS is sending out invalid requests for voxel data. It is requesting data that is not available (even when it is _declared_ to not be available!), resulting in `404`s.

### Reproduction steps

- Open the "Voxels" Sandcastle at https://sandcastle.cesium.com/index.html?src=Voxels.html
- Select "Box - 3D Tiles" from the menu (or any other "3D Tiles" flavor, FWIW)
- Look at the console. It will result in several messages
  `GET https://sandcastle.cesium.com/SampleData/Cesium3DTiles/Voxel/VoxelBox3DTiles/tiles/1/1/0/1.json 404 (Not Found)`

I started zooming into that, and _think_ that it is likely related to this line: https://github.com/CesiumGS/cesium/blob/6c462b7c02ed81bbc5475b03b03fec9884ab3530/packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js#L457 : It is checking the **tile** availability with `tileIsAvailableAtCoordinates`, and based on that, it is requesting **content** with `getVoxelContent` (but that's only a hunch, and has to be confirmed).

(A similar issue was fixed via https://github.com/CesiumGS/cesium/commit/f1b5f8dbd9ececb4fe5a4f8427cbfed15e831168, but I don't know in how far this is related)

This came up via the forum at https://community.cesium.com/t/3d-tiles-1-1-voxel/32239  

### Sandcastle example

See above

### Environment

Browser: Any
CesiumJS Version: 1.117
Operating System: Any

**Comment by ggetz** [comment]
@jjhembd Could you please triage?

**Comment by ggetz** [comment]
Hi, @jjhembd – Just a reminder to please take a look and triage this issue.

**Comment by bertt** [comment]
I'm seeing a similar issue in a quadtree of 2 levels of voxels (CesiumJS 131).

**Issue** see: https://bertt.github.io/cesium_issues/voxels_quadtree_404/issue/

Subtree file has '10110000' as Tile Availability,  '00110000' as Content Availability (there are 0 content tiles on level 0, and 2 content files on level 1). This subtree file works correctly in a mesh environment.

Cesium requests wrongly 0.0.0.glb and stops requesting the tiles on level 1.

**Workaround** is adding a 0.0.0.glb file  (see https://bertt.github.io/cesium_issues/voxels_quadtree_404/workaround/), now 0.0.0.glb is requested and 2 tiles on level 1.

<img width="1880" height="1443" alt="Image" src="https://github.com/user-attachments/assets/fd7b15af-a7a8-42dd-8809-221ee1ff62f8" />

**Comment by javagl** [comment]
Indeed, it looks like this is still an issue. From _quickly_ zooming into the new state, then the reason _still_ looks like the same that I suspected in the first post here. Specifically, the part at https://github.com/CesiumGS/cesium/blob/668c2050e19756a0effbd0ddaa613a350dbf30b3/packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js#L741 is requesting _content_, but seems to be checking the _tile_ availability (and not the _content_ availability). 

(The fact that it did not happen in the sandcastle is likely caused by differences in the data structures: If - roughly speaking - every tile has content in that example data, then the issue will not come up...)

**Comment by jjhembd** [comment]
Thanks @bertt and @javagl for the new info. It looks like we need to update that `isAvailable` check in `Cesium3DTilesVoxelProvider` to also check for content availability.
